packages/core/src/logger.ts

```diff
@@ -0,0 +1,41 @@
+import * as colorette from 'colorette';
+export { options as colorOptions } from 'colorette';
+
+import { isBrowser } from './utils';
+
+const identity = <T>(value: T): T => value;
+
+export const colorize = new Proxy(colorette, {
+  get(target: typeof colorette, prop: string): typeof identity {
+    if (isBrowser()) {
+      return identity;
+    }
+
+    return (target as any)[prop];
+  },
+});
+
+type StdArgs = [string | Uint8Array];
+
+function stderr(...args: StdArgs) {
+  process.stderr.write(...args);
+}
+
+function stdout(...args: StdArgs) {
+  process.stdout.write(...args);
+}
+class Logger {
+  info(...args: StdArgs) {
+    return isBrowser() ? console.log(...args) : stdout(...args);
+  }
+
+  warn(...args: StdArgs) {
+    return isBrowser() ? console.warn(...args) : stderr(...args);
+  }
+
+  error(...args: StdArgs) {
+    return isBrowser() ? console.error(...args) : stderr(...args);
+  }
+}
```

Comment:
Maybe we want to apply a default colorize here to enforce consistency?

Reply:
We have 2 places in `openapi-core` where warnings are shown. Would it be ok if I change the style of these messages from black or white (depending on the system preferences of your terminal) to yellow (in case of warnings)?
Was:

<img width="1389" alt="image" src="https://user-images.githubusercontent.com/22976080/185784384-2e7695e7-7a42-42e2-93a0-fbd493581795.png">

Now: 

<img width="1404" alt="image" src="https://user-images.githubusercontent.com/22976080/185784334-905a302b-c3c9-4c88-aca8-e82db78411e3.png">